app/models/concerns/environment.rb

```diff
@@ -0,0 +1,15 @@
+require 'active_support/concern'
+
+module Environment
+  extend ActiveSupport::Concern
+
+  STAGING = 'staging'.freeze
```

Comment:
By spec, we are not limited to these 3 envs. A user is allowed to set a FAQ with env=potato if they so decide. It seems to me that these vars do not enforce a limitation on what types of values an env can have, but I'm wondering if it might be confusing anyways..